diff --git a/Makefile b/Makefile
index 3a00e05..e018aa1 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,10 @@
-CC=			gcc
+
 #CC=			clang --analyze
 CFLAGS=		-g -Wall -Wno-unused-function -O3
 CXXFLAGS=	-g -Wall -Wno-unused-variable -O3
 WRAP_MALLOC=-DUSE_MALLOC_WRAPPERS
 AR=			ar
-DFLAGS=		-DHAVE_PTHREAD $(WRAP_MALLOC)
+DFLAGS=		-DHAVE_PTHREAD $(WRAP_MALLOC) $(LDFLAGS)
 LOBJS=		utils.o kthread.o kstring.o ksw.o bwt.o bntseq.o bwa.o bwamem.o bwamem_pair.o bwamem_extra.o malloc_wrap.o \
 			intel_ext.o intel_opt/fast_extend_engine.o intel_opt/extend_vec.o \
 			QSufSort.o bwt_gen.o rope.o rle.o is.o bwtindex.o
@@ -13,7 +13,7 @@ AOBJS=		bwashm.o bwase.o bwaseqio.o bwtgap.o bwtaln.o bamlite.o \
 			bwtsw2_core.o bwtsw2_main.o bwtsw2_aux.o bwt_lite.o \
 			bwtsw2_chain.o fastmap.o bwtsw2_pair.o
 PROG=		bwa
-INCLUDES=	
+INCLUDES=-I${PREFIX}/include
 LIBS=		-lm -lz -lpthread
 SUBDIRS=	.
 
@@ -125,6 +125,6 @@ rope.o: rle.h rope.h
 utils.o: utils.h ksort.h malloc_wrap.h kseq.h
 ed_intrav.o: ed_intrav64.h ed_intrav64x2.h ed_intrav.h ed_intravED.h
 ed_intrav.o: ed_fine.h
-intel_opt/fast_extend_engine.o: intel_opt/fast_extend.h 
+intel_opt/fast_extend_engine.o: intel_opt/fast_extend.h
 intel_opt/extend_vec.o: intel_opt/extend_vec128.h intel_opt/extend_vec128x2.h intel_opt/extend_vec256.h intel_opt/extend_vec256x2.h
 intel_ext.o: intel_ext.h
